nilearn/plotting/img_plotting.py

```diff
@@ -780,6 +782,9 @@ def plot_prob_atlas(maps_img, anat_img=MNI152TEMPLATE, view_type='auto',
             view_type = 'continuous'
 
     if threshold is None:
+        # selecting level to use in all three display types
+        threshold = max(threshold, 1e-6)
```

Comment:
Will always return 1e-6 in Python 2.7 and will fail in Python 3.

`threshold=1e-6` is what you wanted, I think!